test/env

```diff
@@ -199,6 +201,7 @@ if [ "$containerize" ]; then
 	 --env PROMPT_DIRTRIM="$ixdir" \
 	 --env PS1="$(build_ps1)" \
 	 --expose 9001-9010 --publish-all \
+	 -v "$HOME/.ash_history":/root/.ash_history \
```

Comment:
Is it really good to get $HOME/.ash_history, that may contain the history from the host system? Isn't something infix specific better: $HOME/.infix/ash_history

Reply:
Hmm, good point.  Iirc we have some other file/dir that we save to `$HOME` as well ... maybe @wkz remembers?

Reply:
The python venv is stored in `~/.infix-test-venv`, maybe that's the one you're thinking of?

IIRC, it takes a while to compose it, so I decided to cache it there to save some time. But it might be better to move that as well :man_shrugging:

Reply:
Ah, of course, thanks!  I'll look into moving that as well, in under a common `~/.infix/`